storages/memory-storage/src/lib.rs

```diff
@@ -29,6 +29,16 @@ pub struct MemoryStorage {
 
 #[async_trait(?Send)]
 impl Store for MemoryStorage {
+    async fn fetch_all_schemas(&self) -> Result<Vec<Schema>> {
+        let mut schemas = self
+            .items
+            .iter()
+            .map(|(_, item)| item.schema.clone())
+            .collect::<Vec<_>>();
+        schemas.sort_by(|a, b| a.table_name.cmp(&b.table_name));
```

Comment:
I feel like it would be better to keep `internal sort_by` as it did rather than external `ORDER BY`

Reply:
Agree